bucket/tla-plus-toolbox.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.7.1",
+    "description": "IDE for the TLA+ tools.",
+    "homepage": "http://lamport.azurewebsites.net/tla/toolbox.html",
+    "license": "MIT",
+    "extract_dir": "toolbox",
+    "url": "https://github.com/tlaplus/tlaplus/releases/download/v1.7.1/TLAToolbox-1.7.1-win32.win32.x86_64.zip",
+    "hash": "sha1:c004561e404743fbb83608229380223a1207a8ec",
+    "bin": [
+        "toolbox.exe",
+        [
+            "tla2tools.jar",
+            "tlc"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "toolbox.exe",
+            "TLA+ Toolbox"
+        ]
+    ],
+    "suggest": {
+        "pretty printer": [
+            "miktex",
+            "latex"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/tlaplus/tlaplus"
+    },
```

Comment:
```suggestion
    "checkver": {
        "github": "https://github.com/tlaplus/tlaplus",
        "regex": "TLAToolbox-([\\d.]+)-win32.win32.x86_64.zip"
    },
```

Reply:
Now it fails the checkver :(

Reply:
Expected, the latest release does not contain win binary.